### PR TITLE
[wip] Add q/a on verified group

### PR DESCRIPTION
**Pull request by ralphtheninja**
A friend of mine asked me and thought this should be in the FAQ.

### CODE REVIEW

**Comment by r10s** [comment]
thanks for the pr and thanks to your friend for caring.
however, i think we should wait until merging this, until the "verified groups" are at least part of the official f-droid version; is currently not 100% clear when this will happen.

**Review by r10s — COMMENTED** [review]
as mentioned, i would wait until verified groups are part of the "official" version.

File: en/help.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ Other messages do not appear automatically. You can see the other messages in th
 - Every group member has the **same rights** as any other. For this reason every one can delete every member or add new ones.
 - To add or delete members, click on the group name in the chat.
 
+## What is a verified group?
+
+- A verified group is a group where each member of the group has been verified by another member out of band. A verified group also enforces encryption. This helps protecting against active attacks.
```
Comment (hpk42) [inline en/help.md]:
```suggestion
- A "verified group" is a labs feature which you can enable in advanced settings.  By adding one another through QR-code scans a group of "verified" members can talk with each other.  It enforces e2e group encryption that is safe against provider or network attacks.  However, some user-tests and feedback suggests that UX and implementation changes makes sense.  Some current discussions (Dec 2018) can be found here: https://countermitm.readthedocs.io/en/latest/new.html

**Comment by r10s** [comment]
cool, from my side, this can be merged now :)

**Review by r10s — APPROVED** [review]

**Pull request merged**